Assets/LeapMotion/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -524,13 +524,60 @@ public class RuntimeGizmoDrawer {
     /// <summary>
     /// Draws a wire gizmo circle at the given position, with the given normal and radius.
     /// </summary>
-    public void DrawWireCirlce(Vector3 center, Vector3 direction, float radius) {
+    public void DrawWireCircle(Vector3 center, Vector3 direction, float radius) {
       PushMatrix();
       matrix = Matrix4x4.TRS(center, Quaternion.LookRotation(direction), new Vector3(1, 1, 0)) * matrix;
       DrawWireSphere(Vector3.zero, radius);
       PopMatrix();
     }
 
+    /// <summary>
+    /// Draws a wire gizmo capsule at the given position, with the given start and end points and radius.
+    /// </summary>
+    public void DrawWireCapsule(Vector3 start, Vector3 end, float radius) {
+      Vector3 up = (end - start).normalized * radius;
+      Vector3 forward = Vector3.Slerp(up, -up, 0.5F);
+      Vector3 right = Vector3.Cross(up, forward).normalized * radius;
+
+      float height = (start - end).magnitude;
+      float distToCap = Mathf.Max(0, (height * 0.5F) - radius);
+      Vector3 middle = (end + start) * 0.5F;
+
+      start = middle + ((start - middle).normalized * distToCap);
+      end = middle + ((end - middle).normalized * distToCap);
+
+      // Radial circles
+      DrawLineWireCircle(start, up, radius);
+      DrawLineWireCircle(end, -up, radius);
+
+      // Sides
+      DrawLine(start + right, end + right);
+      DrawLine(start - right, end - right);
+      DrawLine(start + forward, end + forward);
+      DrawLine(start - forward, end - forward);
+
+      // Endcaps
+      DrawWireArc(start, right, forward, radius, 26, 13);
+      DrawWireArc(start, forward, -right, radius, 26, 13);
+      DrawWireArc(end, right, -forward, radius, 26, 13);
+      DrawWireArc(end, forward, right, radius, 26, 13);
+    }
+
+    private void DrawLineWireCircle(Vector3 center, Vector3 normal, float radius) {
+      DrawWireArc(center, normal, Vector3.Slerp(normal, -normal, 0.5F), radius, 26, 26);
+    }
+
+    private void DrawWireArc(Vector3 center, Vector3 normal, Vector3 radialStartDirection, float radius, int numCircleSegments = 26, int numSegmentsToDraw = 26) {
```

Comment:
I think that instead of passing in two segment arguments, it would be better to pass in first a 'percent filled' argument that defaults to 1.0f, followed by the 'numSegments' argument that has some default that can draw reasonably accurate circles.  

In this way, a user can draw a specific percentage of a circle without needing to do math, or know what the default number of segments are.  They could also up the resolution of the circle without needing to change the number of segments to draw.